pages/maps.js

```diff
@@ -74,7 +71,7 @@ const SwitchText = styled.p`
 const Header = styled.div`
```

Comment:
Plz move styles to a different file

Reply:
where would you move the styling when it is for a page since it isnt inside a component folder?

Reply:
You could create a Header component and import it into the maps.js page. this also creates a higher seperation of concern between logic and renedering